src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/update/SetOperation.java

```diff
@@ -73,27 +74,47 @@ public boolean shouldApplyIf(boolean isInsert) {
   }
 
   @Override
-  public boolean updateDocument(ObjectNode doc) {
+  public UpdateOperationResult updateDocument(ObjectNode doc) {
```

Comment:
method updateDocument signature has been changed in UpdateOperation interface, so as you can see all updateOperations implementation class have updateDocument method modified.
The only difference is that we warp update result into UpdateOperationResult, which has boolean flag modified as always and a potential EmbeddingUpdateOperation.